health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/AssumptionsList.js

```
@@ -0,0 +1,60 @@
+import React from 'react'
+import { Card } from '@egovernments/digit-ui-components';
+import HeaderComp from './HeaderComp';
+import { useTranslation } from 'react-i18next';
+
+const AssumptionsList = ({ customProps }) => {
+    const {t}=useTranslation();
+    const assumptionValues= customProps?.sessionData?.HYPOTHESIS.Assumptions?.assumptionValues
+
+    let dic = {};
```

Biome

[error] This let declares a variable that is only assigned once.

'dic' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)